src/pilot/assign_role.py

```diff
@@ -406,39 +411,50 @@ def define_storage_logical_disks(drac_client, raid_controller_name):
             "configuration")
         return None
 
-    os_logical_disk = define_storage_operating_system_logical_disk(
-        raid_cntlr_physical_disks, drac_client, raid_controller_name)
+    boss_controller = [cntrl for cntrl in raid_controllers
+                       if drac_client.is_boss_controller(cntrl)]
+    if boss_controller and check_cntlr_physical_disks_len(
+         raid_cntlr_physical_disks[boss_controller[0]]):
+        os_logical_disk = define_storage_operating_system_logical_disk(
+            raid_cntlr_physical_disks[boss_controller[0]],
+            drac_client, boss_controller[0])
+    else:
+        raid_controller = [cntrl for cntrl in raid_controllers
+                           if len(raid_cntlr_physical_disks[cntrl]) >= 2]
+        if raid_controller:
+            os_logical_disk = define_storage_operating_system_logical_disk(
+                raid_cntlr_physical_disks[raid_controller[0]],
+                drac_client, raid_controller[0])
+        else:
+            LOG.critical("For configuring RAID 1 needs at least two drives")
```

Comment:
Can you change the above message to:
"At least 2 drives controlled by the same RAID controller are needed to configure a RAID 1"